src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -108,7 +108,7 @@ class="tribe-events-calendar-month__multiday-event-bar-featured-icon tribe-commo
 						></em>
 					<?php endif; ?>
 					<h3 class="tribe-events-calendar-month__multiday-event-bar-title tribe-common-h8">
-						<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+						<?php echo $event->title; ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$event'.

<sub>hash: 9596739896ef8c429ec88f1a1d0564e7</sub>